evalml/pipelines/components/transformers/encoders/onehot_encoder.py

```diff
@@ -72,7 +71,6 @@ def __init__(self,
         self._encoder = None
         super().__init__(parameters=parameters,
```

Comment:
Can we add `random_seed` to the docstrings for parameters 👀

Reply:
Done!